scripts/modules/ai-services.js

```diff
@@ -12,9 +11,10 @@ import { startLoadingIndicator, stopLoadingIndicator } from './ui.js';
 // Load environment variables
 dotenv.config();
 
-// Configure Anthropic client
-const anthropic = new Anthropic({
-  apiKey: process.env.ANTHROPIC_API_KEY,
+// Configure OpenRouter client
+const openrouter = new OpenAI({
+  apiKey: process.env.OPENROUTER_API_KEY,
+  baseURL: "https://openrouter.ai/api/v1"
 });
```

Comment:
we're not adding openrouter integration just yet, this is out of scope of this PR, please remove and suggest in another PR, lets keep the PRs scoped